provision/testdata/flavor_calico-3.23.2_base_case.apic.txt

```diff
@@ -1,4 +1,13 @@
-/api/mo/uni/tn-test/out-calico-l3out-fsvi-vlan-13.json
+/api/mo/uni/tn-common/ctx-calico_vrf.json
+{
+    "fvCtx": {
+        "attributes": {
+            "name": "calico_vrf",
+            "annotation": "orchestrator:aci-containers-controller"
+        }
+    }
+}
+/api/mo/uni/tn-common/out-calico-l3out-fsvi-vlan-13.json
 {
```

Comment:
why is Calico expected output changing?

Reply:
fix-testdata.sh script is updating this.

Reply:
are you using the script from the master branch?

Reply:
yes.

Reply:
fix-testdata.sh scrip is unnecessary updating below files, which is not required(with or without update test passes) 

modified:   provision/testdata/flavor_calico-3.23.2.apic.txt
modified:   provision/testdata/flavor_calico-3.23.2_base_case.apic.txt
modified:   provision/testdata/flavor_calico-3.23.2_tar/custom_resources_aci_calico.yaml